src/extension/app/spectrum-2.css.js

```diff
@@ -96,46 +76,91 @@ export const spectrum2 = css`
+    --spectrum2-action-button-selected-dark: #DADADAFF;
+    --spectrum2-background-color-negative-light: #D73220;
+    --spectrum2-background-color-negative-dark: #CD2E1D;
+    --spectrum2-foreground-color-negative-light: #D73220;
+    --spectrum2-foreground-color-negative-dark: #FC432E;
+    --spectrum2-menu-item-background-color-hover-light: rgba(0, 0, 0, 0.05);
+    --spectrum2-menu-item-background-color-hover-dark: rgba(255, 255, 255, 0.05);
+
+    /* Sidekick theme tokens */
+    --sidekick-max-width: 640px;
+    --sidekick-background-light: #FFFFFFCC;
+    --sidekick-background-dark: #222222CC;
+    --sidekick-background-layer1-dark: #1B1B1B;
+    --sidekick-background-layer1-light: #F8F8F8;
+    --sidekick-background-layer2-dark: #2C2C2C;
+    --sidekick-background-layer2-light: #E9E9E9;
+    --sidekick-background-pasteboard-dark: #111111;
+    --sidekick-background-pasteboard-light: #ffffff;
+    --sidekick-border-color-dark: #393939;
+    --sidekick-border-color-light: #DADADA;
+    --sidekick-backdrop-filter: blur(24px);
+    --sidekick-box-shadow:
+      0px 0px 3px 0px rgba(0, 0, 0, 0.12),
+      0px 3px 8px 0px rgba(0, 0, 0, 0.04),
+      0px 4px 16px 0px rgba(0, 0, 0, 0.08);
 
+    /* Radius tokens */
     --spectrum2-small-border-radius: 4px;
     --spectrum2-default-border-radius: 8px;
     --spectrum2-medium-border-radius: 10px;
     --spectrum2-large-border-radius: 12px;
     --spectrum2-xlarge-border-radius: 14px;
     --spectrum2-xxlarge-border-radius: 16px;
+
+    /**
+     * SPECTRUM ALIAS TOKENS
+     */
+    --spectrum2-background-color-negative: var(--spectrum2-background-color-negative-dark);
+    --spectrum2-foreground-color-negative: var(--spectrum2-foreground-color-negative-dark);
+    --spectrum2-color-positive: var(--spectrum2-color-positive-dark);
+    --spectrum2-action-button-selected: var(--spectrum2-action-button-selected-dark);
+
     --spectrum2-form-input-border-radius: var(--spectrum2-medium-border-radius);
     --spectrum2-dialog-border-radius: var(--spectrum2-xxlarge-border-radius);
-    --spectrum2-sidekick-border-radius: var(--spectrum2-large-border-radius);
 
+    /**
+     * SIDEKICK ALIAS TOKENS
+     */
     --spectrum2-sidekick-background: var(--sidekick-background-dark);
     --spectrum2-sidekick-layer-1: var(--sidekick-background-layer1-dark);
     --spectrum2-sidekick-layer-2: var(--sidekick-background-layer2-dark);
     --spectrum2-sidekick-background-pasteboard: var(--sidekick-background-pasteboard-dark);
     --spectrum2-sidekick-border-color: var(--sidekick-border-color-dark);
     --spectrum2-sidekick-menu-item-background-color-hover: var(--spectrum2-menu-item-background-color-hover-dark);
-    --spectrum2-sidekick-backdrop-filter: blur(24px);
-    --spectrum2-sidekick-box-shadow:
-      0px 0px 3px 0px rgba(0, 0, 0, 0.12),
-      0px 3px 8px 0px rgba(0, 0, 0, 0.04),
-      0px 4px 16px 0px rgba(0, 0, 0, 0.08);
+    --spectrum2-sidekick-border-radius: var(--spectrum2-large-border-radius);
```

Comment:
Does the `spectrum2-` prefix in sidekick tokens have any purpose?

Reply:
It's always good to namespace variables since they penetrate the shadow DOM. The double name spacing here might not be needed though. I will log an issue to look at this in a future PR since the change will touch a lot of files.